rtlib/include/rt/LandmarkIO.hpp

```diff
@@ -5,6 +5,7 @@
 #include "rt/ImageTypes.hpp"
 #include "rt/LandmarkRegistration.hpp"
 
+namespace fs = boost::filesystem;
```

Comment:
This definitely makes sense. But is there anyway​ that we can make the namespace alias scope to just the class? Seems a little strange to have a private member variable in the header using boost::filesystem but in the cpp using fs::